Flaky test: 'tests selecting items in overflow menu' (SideNavigation.cy.tsx)

## Summary

The Cypress test `Side Navigation interaction > tests selecting items in overflow menu` (`packages/fiori/cypress/specs/SideNavigation.cy.tsx:994`) fails intermittently on CI and is blocking PR merges.

## Failing assertion

```
AssertionError: Timed out retrying after 4000ms: expected
'<ui5-side-navigation-item#ui5wc_282-sn-overflow-item.ui5-sn-item-overflow>'
to be 'focused'
  at Context.<anonymous> (SideNavigation.cy.tsx:1048:4)
```

Example failing run: https://github.com/UI5/webcomponents/actions/runs/26900555688/job/79351448244?pr=13634

## Status

Test temporarily skipped via PR #13637 to unblock CI. This issue tracks re-enabling it once the underlying flakiness is resolved.

--- a/packages/fiori/cypress/specs/SideNavigation.cy.tsx
+++ b/packages/fiori/cypress/specs/SideNavigation.cy.tsx
@@ -991,8 +991,7 @@ describe("Side Navigation interaction", () => {
 		cy.get("@selectionChangeHandler").should("have.been.calledOnce");
 	});
 
-	// Skipped: flaky on CI — blocks merges. Tracked separately.
-	it.skip("tests selecting items in overflow menu", () => {
+	it("tests selecting items in overflow menu", () => {
 		cy.mount(
 			<SideNavigation style="height: 200px" id="sideNav" collapsed={true}>
 				<SideNavigationItem icon={home} text="Home"></SideNavigationItem>
@@ -1023,14 +1022,14 @@ describe("Side Navigation interaction", () => {
 			.as("overflowMenu");
 
 		cy.get("@overflowMenu")
-			.should("be.visible");
+			.ui5MenuOpened();
 
 		cy.get("@overflowMenu")
 			.find("[ui5-navigation-menu-item][text='Home 6']")
 			.realClick();
 
 		cy.get("@overflowMenu")
-			.should("not.be.visible");
+			.ui5MenuClosed();
 
 		cy.get("[ui5-side-navigation-item][text='Home 6']")
 			.should("be.focused");
@@ -1039,11 +1038,14 @@ describe("Side Navigation interaction", () => {
 			.realClick();
 
 		cy.get("@overflowMenu")
-			.should("be.visible");
+			.ui5MenuOpened();
 
 		cy.get("@overflowMenu")
 			.find("[ui5-navigation-menu-item][text='Home 7']")
 			.realClick();
+
+		cy.get("@overflowMenu")
+			.ui5MenuClosed();
 
 		cy.get("@itemOverflow")
 			.should("be.focused");

